client/components/chart/index.jsx

```diff
@@ -3,62 +3,56 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
 import { some, noop, throttle } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import BarContainer from './bar-container';
-import observe from 'lib/mixins/data-observe';
 import touchDetect from 'lib/touch-detect';
 
-module.exports = React.createClass( {
-	displayName: 'ModuleChart',
+class ModuleChartExport extends React.Component {
+	state = {
```

Comment:
Would it be better to follow React's documentation for setting the initial state using `this.state` in the constructor?

See: https://facebook.github.io/react/docs/react-without-es6.html#setting-the-initial-state

Reply:
We are using both styles these days. I think this is a new ES6+ syntax which we recently discovered and started using. Actually, we got suggestion to avoid using constructor in one of other PRs yesterday :) So I would say, let's leave it as it is.